extensions-core/lookups-cached-global/src/test/java/org/apache/druid/server/lookup/namespace/cache/JdbcExtractionNamespaceTest.java

```diff
@@ -63,6 +63,9 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
nit: we usually try to avoid static imports, like I see no instances of `import static junit.framework.TestCase.assertEquals` (also, i think we usually use `Assert.assertEquals`) and only 28 for `import static org.junit.Assert.assertTrue` across the codebase

Reply:
fixed